minimal-flows/docker-compose.yaml

```diff
@@ -69,7 +67,7 @@ services:
     ports:
       - "8101:8101/tcp"
       - "8980:8980/tcp"
-      - "10162:1162/udp"
+      - "10162:162/udp"
       - "4729:4729/udp"
```

Comment:
Maybe use the multi protocol listener udp/9999 here. It will allow to send IPFIX, NetFlow v5/v9 and SFlow records.